src/seedu/addressbook/commands/Command.java

```diff
@@ -70,4 +70,15 @@ public int getTargetIndex() {
     public void setTargetIndex(int targetIndex) {
         this.targetIndex = targetIndex;
     }
+
+    /**
+     * Check whether the command mutate the data.
+     *
+     * @return true if the command mutate the data, false if it does not mutate the data
+     *
+     * Default is set to true
+     */
+    public boolean isMutating() {
+        return true;
```

Comment:
We shouldn't assign a default behavior to the parent class even though its generally true/false
E.g. what if we assign `canFly()` to be true by default and we forgot to override it for some animals such as penguin?
result -> suddenly penguin can fly